src/clustering/sharding-sqlserver/SqlSharding.Host/sharding.conf

```diff
@@ -30,12 +30,12 @@ akka{
 
                 event-adapters {        
                     tagger = "SqlSharding.Shared.MessageTagger, SqlSharding.Shared"
-                    #coordinator-migration = "Akka.Cluster.Sharding.OldCoordinatorStateMigrationEventAdapter, Akka.Cluster.Sharding"
+                    coordinator-migration = "Akka.Cluster.Sharding.OldCoordinatorStateMigrationEventAdapter, Akka.Cluster.Sharding"
```

Comment:
Having this line active when only 1.4 is installed will cause a blow-up. Disable it when running on v1.4.